### PR TITLE
Reuse crix

### PR DESCRIPTION
**Pull request by m5imunovic**

### CODE REVIEW

File: safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!--
+SPDX-FileCopyrightText: 2020 callgraph-tool authors. All rights reserved
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Licence and Copyright information in the callgraph-tool
+
+We are doing our best to keep all the licencing and copyright information in place.
```
Comment (henrirosten) [inline safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md:9]: Typo: licencing ==> licensing

File: safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+If you are adding multiple files or files that don't have editable header you can licence the
+whole file by adding the appropriate entry in ./reuse/dep5 file. For example, the copyright
+and license information for entire contents of `doc` directory can be defined as:
+```
+Files: doc/*
+Copyright: 2020 callgraph-tool authors. All rights reserved.
+License: Apache-2.0
+```
+
+Before commiting/pushing the file make sure that the reuse conformance is kept with:
```
Comment (henrirosten) [inline safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md:52]: Typo: commiting ==> committing

File: safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+```
+reuse addheader <NAME_OF_NEW_FILE>\
+    --copyright "callgraph-tool authors. All rights reserved"\
+    --licence Apache-2.0
+    --year <CURRENT_YEAR>\
+    --style <FILE_STYLE>\
+```
+
+If you are adding multiple files or files that don't have editable header you can licence the
+whole file by adding the appropriate entry in ./reuse/dep5 file. For example, the copyright
```
Comment (henrirosten) [inline safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md:44]: "whole file" should be "whole directory"?

File: LICENSES/LLVM-exception.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+LLVM Exceptions to the Apache 2.0 License
```
Comment (henrirosten) [inline LICENSES/LLVM-exception.txt:1]: Why do we add the file LLVM-exception.txt?

This was not in the original crix repository, and I don't see it being referenced anywhere.

File: safety-architecture/tools/callgraph-tool/AUTHORS [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<!--
+SPDX-FileCopyrightText: 2020 callgraph-tool authors. All rights reserved
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# This file contains copyright holder of the callgraph-tool repository
```
Comment (henrirosten) [inline safety-architecture/tools/callgraph-tool/AUTHORS:7]: Should this be changed: 
"contains copyright holder" ==> "contains copyright holders"
already anticipating that there might be more than one copyright holder?

**Comment by henrirosten** [comment]
Please check the comments I added, otherwise I think we are good to go with the merge.

File: safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md (hunk 1 above)
Comment (m5imunovic) [reply safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md:9]: Fixed multiple occurrences, renamed dir and file

File: LICENSES/LLVM-exception.txt (hunk 4 above)
Comment (m5imunovic) [reply LICENSES/LLVM-exception.txt:1]: This is an artifact of an idea that I had for a brief period of time. Will be removed.

**Pull request merged**